### PR TITLE
Support for 'mutable-content'

### PR DESCRIPTION
**Pull request by Mortenest**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,14 +13,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Pushwoosh/pushwoosh-parse-adapter.git"
+    "url": "git+https://github.com/HangAppApS/pushwoosh-parse-adapter.git"
```
Comment (shaders) [inline package.json]: Could you please revert this change to original repository?
Thanks!

File: package.json [hunk 2]
```diff
@@ -13,14 +13,14 @@
...
   },
   "author": "Deonis Peretyagin <karudo@gmail.com> (https://github.com/karudo)",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/Pushwoosh/pushwoosh-parse-adapter/issues"
+    "url": "https://github.com/HangAppApS/pushwoosh-parse-adapter/issues"
```
Comment (shaders) [inline package.json]: Could you please revert this change to original repository?
Thanks!

File: package.json [hunk 3]
```diff
@@ -13,14 +13,14 @@
...
   },
-  "homepage": "https://github.com/Pushwoosh/pushwoosh-parse-adapter#readme",
+  "homepage": "https://github.com/HangAppApS/pushwoosh-parse-adapter#readme",
```
Comment (shaders) [inline package.json]: Could you please revert this change to original repository?
Thanks!

**Comment by Mortenest** [comment]
Hi Shaders, 

I have reverted these urls now.

Thanks

**Comment by shaders** [comment]
Merged. Thanks!
